native/explorer/src/series.rs

```diff
@@ -1659,12 +1659,21 @@ pub fn s_second(s: ExSeries) -> Result<ExSeries, ExplorerError> {
 }
 
 #[rustler::nif(schedule = "DirtyCpu")]
-pub fn s_strptime(s: ExSeries, format_string: &str) -> Result<ExSeries, ExplorerError> {
+pub fn s_strptime(
+    s: ExSeries,
+    format_string: Option<&str>,
+    precision: Option<&str>,
```

Comment:
We could use `ExTimeUnit` enum (that is declared inside the `src/datatypes/ex_dtypes.rs`) here - instead of `&str`. I think we could also implement the `TryFrom` trait for this conversion. WDYT?

Reply:
I gave it a shot!

[2953b52](https://github.com/elixir-explorer/explorer/pull/795/commits/2953b52748f81aef926251e0513387f9b0de3303)

I cargo-culted it. LMK if it wasn't right.

Reply:
Nice, it's correct! Thanks!